src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionIndex.cs

```diff
@@ -74,7 +74,7 @@ internal MotionIndex(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, c
         /// <summary>
         /// The id of ModelNode. If you want to apply to all ModelNodes who has BlendShape string, let this value as null.
```

Comment:
"let this value as null" -> "assign null", "keep the null as the property value"
Please look around for "id" in lowercase in the text   `git show | grep " id"` and change them to ID

Reply:
Fix done.